delfin/api/v1/storages.py

```diff
@@ -162,6 +162,17 @@ def sync(self, req, id):
                 storage['id'],
                 subclass.__module__ + '.' + subclass.__name__)
 
+    def perf_collect(self, storage_id, interval, is_history, resource):
```

Comment:
Purpose of this function ? who invokes this?
I hope this should be based on router , for performance register ..

Reply:
yes... this is invoked by scheduler